### DISCUSSION TITLE
如何支持markdown 和数学公式的编辑与展示

### DISCUSSION THREAD

**Discussion by quhongli123** [wangeditor-next/wangEditor-next#322] [General]
## 问题描述

*如何支持markdown 和数学公式的编辑与展示*

## wangEditor 版本  5

*请输入内容……*

## 是否查阅了文档 ？

（文档链接 [https://cycleccc.github.io/docs/](https://cycleccc.github.io/docs/) ）

*是*

## 最小成本的复现步骤

（请告诉我们，如何**最快的**复现该问题？）

- 步骤一
- 步骤二
- 步骤三

**Comment by cycleccc** [comment]
Markdown 有对应插件 https://github.com/cycleccc/wangEditor-next-plugin-md
数学公式 的 插件我还没迁移，这周末有空回迁以下。

使用方式看官方文档及对应的插件文档就可以。

**Reply by quhongli123** [reply]
大佬   希望数学公式希望可以在表格里展示/编辑 [合十]

**Reply by cycleccc** [reply]
markdown 和 formula 都加到 packages 里 并发包了，测过了可以正常使用，数学公式可以在表格里展示和，table单元格中编辑的弹窗还没做，暂时是要删除原来的再加入如以下视频所示

https://github.com/user-attachments/assets/afe52ffb-5648-41cb-bce7-01f7e568a2a0

**Reply by quhongli123** [reply]
感谢大佬

**Comment by cycleccc** [comment]
Markdown 与公式均已有对应插件；公式插件已迁移并持续发布。本讨论现关闭；如有接入问题请提供最小复现。
